### ISSUE TITLE
Serenity doesn't count tests when using SerenityParameterizedRunner without @Managed

### ISSUE THREAD

**Issue by dsmt** [serenity-bdd/serenity-core#179]
Sometimes you don't need webdriver. When it is omitted with SerenityRunner.class everything is Ok, but with SerenityParameterizedRunner.class the amount of tests in the report is always 1

    @Managed
    public WebDriver webdriver;

**Comment by YamStranger** [comment]
Hello @dsmt!
Thanks for reporting us. 

This issue should fixed under 1.1.22-rc.5 release. 

Please close this issue, if all works for you. 

Thanks!!!

**Comment by dsmt** [comment]
@YamStranger, many thanks for your work!
I have checked this issue but unfortunately the defect can be reproduced under **1.1.22-rc.5** while [180](https://github.com/serenity-bdd/serenity-core/issues/180) has been fixed

**Comment by YamStranger** [comment]
Hello @dsmt. 

Lets check that I get your point. For such purpose lets use our demo project. 

To investigate your problem I done next:

1) Clone it: [git@github.com:serenity-bdd/serenity-demos.git](git@github.com:serenity-bdd/serenity-demos.git)
2) navigate to  junit-parameterized-tests. 
3) update version of serenity in gradle.build in this directory to use current version:
```
repositories {
    mavenLocal()
    jcenter()
}

ext{
    serenity_version = "1.1.22-rc.7"
}

buildscript {
    repositories {
        mavenLocal()
        jcenter()
    }
    dependencies {
        classpath("net.serenity-bdd:serenity-gradle-plugin:1.1.22-rc.7")
    }
}

apply plugin: "java"
apply plugin: "eclipse"
apply plugin: "idea"
apply plugin: "net.serenity-bdd.aggregator"

dependencies {
    compile localGroovy()
    compile "net.serenity-bdd:serenity-core:$serenity_version"
    compile "net.serenity-bdd:serenity-junit:$serenity_version"
    compile("net.serenity-bdd:serenity-rest-assured:$serenity_version")
    compile 'org.slf4j:slf4j-simple:1.7.12'
    compile 'org.codehaus.groovy:groovy-all:2.3.10'
    compile 'junit:junit:4.12'
    compile 'com.googlecode.lambdaj:lambdaj:2.3.3'
    compile 'org.assertj:assertj-core:3.1.0'
}

test {
    testLogging {
        showStandardStreams = true
    }
    /* Pass all system properties: */
    systemProperties System.getProperties()
}

task copyDeps(type: Copy) {
    from configurations.runtime
    into project.projectDir.path+"/lib"
}

gradle.startParameter.continueOnFailure = true
```
4) run in this dir next command
```
gradle clean test aggregate
```

After this build executed, all reports should be under target dir. 

According to current configuration next class executed:
```
@RunWith(SerenityParameterizedRunner.class)
@UserStoryCode("US01")
@Concurrent
public class SampleTest {
    @TestData
    public static Collection<Object[]> testData() {
        return Arrays.asList(new Object[][]{{1}, {2}, {3}});
    }

    @Steps
    TestSteps steps;
    private final int value;

    public SampleTest(int value) {
        this.value = value;
    }

    @Test
    public void shouldExecuteThisTest() {
        steps.initialization();
        steps.when_example_action_for(this.value);
        steps.then_example_result_should_be(this.value);
    }

    @Test
    public void shouldExecuteThisTestToo() {
        steps.initialization();
        steps.when_example_action_for(this.value);
        steps.then_example_result_should_be(this.value);
    }
}
```
This class contains two test and some amount of steps. All of them should be in report. 

And here is page with tests and with steps
![two_tests](https://cloud.githubusercontent.com/assets/1667699/11285021/fc49e41c-8f15-11e5-8efd-9a6bfd81f978.png)

![two_tests_step_1](https://cloud.githubusercontent.com/assets/1667699/11285026/026619e2-8f16-11e5-967e-1f393d2ec0e4.png)

![two_tests_step_2](https://cloud.githubusercontent.com/assets/1667699/11285027/026f4bde-8f16-11e5-999a-b7f4933ceca7.png)

@dsmt please help me investigate the problem, I not sure that I can reproduce it.

**Comment by dsmt** [comment]
@YamStranger, it seems that didn't describe the issue clear enough, so I just waste your time =(
Ok, here is the second try. In this issue I'm talking about information in the major block of report. You can see the screenshots which I included this time. Hope they will help. If they not I can do example project.

![](http://take.ms/mm5iA)
![](http://take.ms/l3P9K)
![](http://take.ms/KkWM0)

**Comment by cliviu** [comment]
I think it should be fixed now, please give it a spin :-)

**Comment by YamStranger** [comment]
Hello @dsmt. This fix should be available under [1.1.22-rc.10](https://github.com/serenity-bdd/serenity-core/releases/tag/v1.1.22-rc.10). 

Please check it, and if you fill that this problem resolved - please close this issue. 

Thanks a lot!
